scripts/variantstore/wdl/GvsRescatterCallsetInterval.wdl

```diff
@@ -12,7 +12,7 @@ workflow GvsRescatterCallsetInterval {
     File reference
     File reference_dict
     File reference_index
-    String final_output_gcs_dir
+    String? final_output_gcs_dir
```

Comment:
this gets passed into merge vcfs. do we know whether this works if it is not specified?
$(echo ~{output_directory} | sed 's/\/$//')